### PR TITLE
Edits to harmonise references to the Autonity CLI tool `autcli`.

### LINKED ISSUE #33: Harmonise references to the Autonity CLI tool `autcli`
**Describe the bug**

There is inconsistent terminology used when referring to the Autonity CLI tool `autcli`:

- `aut` CLI
- `autcli`

`aut` is a command, not the tool name which is Autonity CLI, or `autcli` after the tool's GitHub repo name https://github.com/autonity/autcli.

To resolve:

- correct instances of `aut` CLI -> `autcli`.
- rename 'Setup aut CLI' -> 'Setup Autonity CLI'

The "`aut` CLI" issue occurs on the following pages

```console
% grep -lr '`aut` CLI' .
./content/docs/developer/_index.md
./content/docs/node-operators/connect/_index.md
./content/docs/node-operators/_index.md
./content/docs/delegators/_index.md
./content/docs/delegators/bond-stake/_index.md
./content/docs/concepts/client/_index.md
./content/docs/validators/register-vali/_index.md
./content/docs/validators/_index.md
./content/docs/account-holders/submit-trans-autcli/_index.md
./content/docs/_index.md
```
**To Reproduce**

Search for above terms to find the inconsistency.

**Device information (please complete the following information):**
n/a

### PR DESCRIPTION
**Pull request by jay-cousins**
Edits to harmonise references to the Autonity CLI tool `autcli` as described in the related issue.

closes #33

### CODE REVIEW

**Comment by aiman** [comment]
LGTM

**Review by aiman — APPROVED** [review]

File: content/docs/developer/_index.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ The steps in this section describe how to set up a local testnet for development
 
 It is assumed that you have:
 
-- [Setup the `aut` CLI](/account-holders/setup-autcli/) tool and have an [account](/account-holders/create-acct/).
+- [Setup the `autcli`](/account-holders/setup-autcli/) tool and have an [account](/account-holders/create-acct/).
```
Comment (aiman) [inline content/docs/developer/_index.md]: Should this be more consistent with the revised page title or linkTitle? E.g.

> \[Setup the Autonity CLI tool](...)

or

> \[Setup Autonity CLI](...)

or perhaps just include the word "tool" in the hyperlink?

**Review by aiman — COMMENTED** [review]
In files:

- `content/docs/account-holders/submit-trans-autcli/_index.md`
- `content/docs/account-holders/unlock-acct/_index.md`
- `content/docs/developer/submit-trans-gov-acct/_index.md`
- `content/docs/reference/api/aut/_index.md`
 
I see:

> Autonity autcli
> Autonity \`autcli\`

Does this require a change? Sounds a bit clunky? For example, could replace with one/more of:

> Autonity CLI
> autcli

Also in `content/docs/account-holders/submit-trans-autcli/_index.md`:

> \[of \`autcli\`](https://github.com/autonity/autcli) 

Perhaps should be:

> of \[\`autcli\`](https://github.com/autonity/autcli) 


?

**Comment by aiman** [comment]
I have revised my review comment.

**Comment by dtebbs** [comment]
I have a bit of an aversion to `autcli` being used as the name of the tool itself.  The command is aut `aut` and the CLI part is really only used to distinguish it from `autonity` (the full node).  We wouldn't make a command `autcli` as that would be truly redundant.  GUIs, although useful for some things, aren't really consistent with what this tools is intended for (namely as the basis for scripting, automation or otherwise calling from other processes) so CLI should be implicit.

Not sure what others think, but if we are harmonizing I'd rather harmonize around the 3 letter command, i.e. "`aut` CLI tool" or "`aut` tool" (where the repo for this tool happens to have the `cli` suffix to distinguish it from `autonity`, but that's just for the repo) and then rethink the actual name at leisure (I've heard suggestions for things like `ask`, `act`, and I'd add `tau`, but I'm sure there are other good candidates).

**Comment by rmsams** [comment]
I'm with @dtebbs on this.. just refer to it as `aut` rather than `autcli` or `aut` CLI tool.

So sentences like: 

> The aut CLI tool should be configured to connect to the local Autonity testnet you have setup.

Become:

> `aut` should be configured to connect to the local Autonity testnet you have setup.

It's universally understood that typewriter text means code/commands, there's no need keep describing what it is everytime we _use_ `aut` to _refer_ to the tool. 

Note: this is a general point. E.g., you wouldn't say "the `gcc` compiler tool", you just say `gcc`.

**Comment by dtebbs** [comment]
Just an addition to this, I agree with just keeping it simple with`aut`, but just mentioning it could also stand for "Autonity Utility Tool".

**Comment by jay-cousins** [comment]
OK, reviewing comments, revising the change in terminology to:

- correct instances of `aut` CLI , `autcli` --> aut.
- rename 'Setup Autonity CLI'' --> 'Setup Autonity Utility Tool (`aut`)
- edit hyperlinks to aut repository (renamed to aut from autcli)

File: content/docs/developer/_index.md (hunk 1 above)
Comment (jay-cousins) [reply content/docs/developer/_index.md]: Correct. Edited to the revised page title. Commit: 1a94c1cac2b5aeb048dff0255286261c390bb78d

**Review by jay-cousins — COMMENTED** [review]
PR updated by commit 1a94c1cac2b5aeb048dff0255286261c390bb78d to address review comments received to date.

Re-requesting review 👍

**Comment by aiman** [comment]
+1 for Autonity Utility Tool

**Review by dtebbs — APPROVED** [review]
LGTM

**Review by lorenzo-dev1 — APPROVED** [review]

**Pull request merged**